### DISCUSSION TITLE
How to debug code in eclipse

### DISCUSSION THREAD

**Discussion by sarveshhiwase** [dbeaver/dbeaver#22524] [DBeaver Development]
## Overview
- I just started exploring dbeaver project, and just read the full documentation on how to setup the development environment on eclipse, I did all the same steps mentioned in doc, but still I couldn't debug via breakpoints setup
- ### Issue
  - I tried setting up the breakpoint on a method that I want to debug, tried performing the UI scenario, but the debugger won't stop.
![Screenshot from 2024-01-21 23-19-49](https://github.com/dbeaver/dbeaver/assets/69983214/2d6ffb35-a62e-4733-9e5b-310c90c62d49)
![Screenshot from 2024-01-21 23-19-56](https://github.com/dbeaver/dbeaver/assets/69983214/c6b97a5f-715f-4390-a3e7-f8ddadfed08d)

@LonwoLonwo Can you please help me on this one?
Also want to know, where are logs that are created by logger located?

**Comment by LonwoLonwo** [comment]
Hello @sarveshhiwase 

Did you try to set your breakpoints inside methods? Like on lines 111/115, etc?
Also, what exactly are you trying to catch? Because proposals are more about the SQL Editor's autocompletion.

Usually, you can find logs in the Console.

**Reply by sarveshhiwase** [reply]
Hello @LonwoLonwo, really thanks for replying.
I tried setting up breakpoints inside methods too, but didn't work.
Now I tried setting up the whole environment again, the debugging just stops after 20 sec.
Can you please give me points to properly set up the environment setup for eclipse? 
I read the documentation and did the full steps again and again, but couldn't succeed.

**Reply by sarveshhiwase** [reply]
@LonwoLonwo I have found my eclipse issue, now it's working fine, I was checking wrong file, sorry for hassle and thanks for replying.

**Reply by LonwoLonwo** [reply]
Ok, thanks for the update!
